### PR TITLE
[CUMULUS-2176] Update GNF Report Styles

### PR DESCRIPTION
**Pull request by dopeters**
Date range moved to same line as header
Added Total Conflict Comparisons
Download report below border
Table legend moved to Collapse component above table

**Summary:** Summary of changes

Addresses [CUMULUS-2176: Reports: GNF View - Update Style & Interaction](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2176)

## Changes

* Added Total Conflict Comparison section w/ description (for both GNF and Inventory)
* Removed report status
* Moved date range to be in line with header
* Made Legend collapsible and added it next to the column filters collapsible button
* Added tooltips for the colored indicators

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: app/src/js/components/Form/_form.scss [hunk 1]
```diff
@@ -62,74 +62,6 @@ select option{
...
-  align-items: flex-end;
-}
-
-.filter-actions {
-  display: flex;
-  justify-content: space-around;
-  flex-grow: 1;
-
-  .form--controls {
-    flex-grow: 1;
```
Comment (dopeters) [inline app/src/js/components/Form/_form.scss:132]: None of these styles are used anywhere

**Comment by jjmccoy** [comment]
@dopeters Looks good! I did notice one thing: When click on 'Show Column Filters' to make a selection it pushes the Table Legend heading and button to the right. 

Screen Shot 2021-01-11 at 11 13 06 AM

**Comment by dopeters** [comment]
@jjmccoy Yeah I had looked into that but was having trouble getting the buttons to stay put. Seems like the `Collapse` component is intentionally designed that way. It does make it so there's room for the legend to expand when both buttons are clicked. Otherwise there would be some overlap, or the legend wouldn't display under its button. Probably could find a workaround, but was having trouble fighting against the styles that come with the component
Screen Shot 2021-01-11 at 11 26 11 AM

One alternative is making the column options display vertically so it doesn't push the button over like so:
Screen Shot 2021-01-11 at 11 25 50 AM

Could also do something to the effect of making the button width bigger so that the area for the column options is wider, thereby allowing us to wrap them. Something like this:
Screen Shot 2021-01-11 at 11 33 30 AM

Let me know your thoughts on those options or if you have any other ideas here!

**Comment by jjmccoy** [comment]
> @jjmccoy Yeah I had looked into that but was having trouble getting the buttons to stay put. Seems like the `Collapse` component is intentionally designed that way. It does make it so there's room for the legend to expand when both buttons are clicked. Otherwise there would be some overlap, or the legend wouldn't display under its button. Probably could find a workaround, but was having trouble fighting against the styles that come with the component
> Screen Shot 2021-01-11 at 11 26 11 AM
> 
> One alternative is making the column options display vertically so it doesn't push the button over like so:
> Screen Shot 2021-01-11 at 11 25 50 AM
> 
> Could also do something to the effect of making the button width bigger so that the area for the column options is wider, thereby allowing us to wrap them. Something like this:
> Screen Shot 2021-01-11 at 11 33 30 AM
> 
@dopeters Oh, I see. I think this workaround would be good. If we make the 'Show/Hide Column Filters' wider then we may need to put the selections into columns or have more padding when they wrap.
> Let me know your thoughts on those options or if you have any other ideas here!

**Comment by dopeters** [comment]
> @dopeters Oh, I see. I think this workaround would be good. If we make the 'Show/Hide Column Filters' wider then we may need to put the selections into columns or have more padding when they wrap.

Thanks for confirming! Just pushed that change

**Comment by jjmccoy** [comment]
@dopeters Nice! Thanks for the updates.

**Review by jjmccoy — APPROVED** [review]
LGTG!

**Pull request merged**